### DISCUSSION TITLE
Why my machine state has only subscribe function?

### DISCUSSION THREAD

**Discussion by SimYunSup** [statelyai/xstate#3189] [Q&A] [answered]
My machine in svelte has only subscribe function, so I can't develop my application.

![image](https://user-images.githubusercontent.com/47362439/161429015-473eac1a-ff74-4a31-9856-befde676a03b.png)

https://codesandbox.io/s/svelte-xstate-error-jb9ui0?file=/Button.svelte

**Comment by Andarist** [comment, marked as answer]
It's a readable store so you need to "subscribe" to it with `$` (as per the Svelte docs [here](https://svelte.dev/docs#component-format-script-4-prefix-stores-with-$-to-access-their-values)). See the fixed codesandbox: https://codesandbox.io/s/svelte-xstate-error-forked-v0rdie?file=/Button.svelte:196-202

**Reply by SimYunSup** [reply]
Thank you for kindly answer. And [Usage with svelte](https://xstate.js.org/docs/recipes/svelte.html) looks good to fix with code using useMachine.
